### ISSUE TITLE
Popover broken on Android

### ISSUE THREAD

**Issue by varand-pez** [tamagui/tamagui#2287]
### Current Behavior

Popover component is breaking the app on Android throwing the following error:

```
 ERROR  Invariant Violation: [1143,"RCTView",{"top":"<<NaN>>","left":"<<NaN>>"}] is not usable as a native method argument
```

### Expected Behavior

Popover works as expected and not break the app. iOS is fine.

### Tamagui Version

```markdown
"tamagui": "^1.89.28",
```

### Platform (Web, iOS, Android)

```markdown
Android
```

### Reproduction

```markdown
here is the popover implementation:

<Popover placement="top" size="$5">
  <Popover.Trigger asChild>
    <Button
      size="$3"
      icon={<Info size="$1" />}
      circular
      chromeless
      color="$orange9"
    />
  </Popover.Trigger>

  <Popover.Content
    borderWidth={1}
    borderColor="$orange9"
    enterStyle={{ y: 15, opacity: 0 }}
    exitStyle={{ y: 15, opacity: 0 }}
    elevate
    p="$2"
    backgroundColor="$gray1"
    animation={[
      'quick',
      {
        opacity: {
          overshootClamping: true,
        },
      },
    ]}
  >
    <Popover.Arrow borderWidth={1} borderColor="$orange9" />
    <XStack ai="center" jc="space-between" gap="$2">
      <Text w="$19">Hello World</Text>
      <Popover.Close asChild>
        <Button size="$3" icon={<Close size="$1" />} circular chromeless />
      </Popover.Close>
    </XStack>
  </Popover.Content>
</Popover>

```
```

### System Info

_No response_

**Comment by varand-pez** [comment]
![Screenshot_1708228536](https://github.com/tamagui/tamagui/assets/47469980/83c85763-84ce-4590-92df-6d3697e11a91)

**Comment by TheEhsanSarshar** [comment]
I made a [PR](https://github.com/floating-ui/floating-ui/pull/2798) that will fix this issue
will be fixed in Tamagui once that's merged
although in general, it's better to adapt to sheet on Android

**Comment by varand-pez** [comment]
Hi @TheEhsanSarshar jan, would you care to share some sample of what you suggested? 

> it's better to adapt to sheet on Android

Thanks.

**Comment by varand-pez** [comment]
with version `1.90.2` the error is gone but the popover is not rendering at correct place, both for iOS and Android.
It appears on right bottom corner

**Comment by TheEhsanSarshar** [comment]
yeah you can use sheet on native, check [doc](https://tamagui.dev/docs/components/popover/1.83.0) example

**Comment by TheEhsanSarshar** [comment]
> with version `1.90.2` the error is gone but the popover is not rendering at correct place, both for iOS and Android. It appears on right bottom corner

ok I'll check that, probably something was broken with onLayout and now it's fixed

**Comment by varand-pez** [comment]
@TheEhsanSarshar any update on this?

**Comment by TheEhsanSarshar** [comment]
merged a fix, things were working fine on `android / ios` so I'll close this issue

**Comment by varand-pez** [comment]
in what version will it appear?
https://github.com/tamagui/tamagui/pull/2426

**Comment by TheEhsanSarshar** [comment]
in next version

**Comment by geoiq-tech-team** [comment]
Hey @TheEhsanSarshar when will that be?

**Comment by varand-pez** [comment]
It was in shipped with version `v1.91.4`
I can confirm that the issue is fixed. 

cc @geoiq-tech-team

**Comment by geoiq-tech-team** [comment]
Appreciate it. Thanks!
